internal/ignition/dummy.go

```diff
@@ -54,6 +54,8 @@ func (g *dummyGenerator) Generate(_ context.Context, installConfig []byte, platf
 			data = kubeconfig
 		} else if fileName == "bootstrap.ign" {
 			data = bootstrapIgnition
+		} else if fileName == "install-config.yaml" {
+			data = string(installConfig)
```

Comment:
If we have this can we remove the logic at the beginning of the function that writes out the install config specifically?

Reply:
My guess is that file was getting overwritten by the default data in this loop.